src/dom-to-image-more.js

```diff
@@ -206,7 +205,6 @@
         options = options || {};
         return toSvg(domNode, options)
             .then(util.makeImage)
-            .then(util.delay(0))
```

Comment:
This was put in a VERY long time ago to fix some yielding issues. It needs to go away.